test/model.test.js

```diff
@@ -10,6 +10,7 @@ var defineModelTestsWithDataSource = require('./util/model-tests');
 var PersistedModel = loopback.PersistedModel;
 
 var describe = require('./util/describe');
+require('./helpers/loopback-testing-helper');
```

Comment:
As I commented before, I don't understand why is this needed, could you please explain?